### DISCUSSION TITLE
load and display protein structure

### DISCUSSION THREAD

**Discussion by Lili-Cao** [chemosim-lab/ProLIF#197] [Q&A]
![image](https://github.com/chemosim-lab/ProLIF/assets/76455307/5d8d765e-bfe4-45f7-9b77-39a06cfc0c30)
I get a lot of warning like WARNING: not removing hydrogen atom without neighbors

the protein_file file are from maestro in pdb format 

u = mda.Universe(protein_file)
protein_mol = plf.Molecule.from_mda(u)
plf.display_residues(protein_mol)

May I ask why the display function cannot show the residues?

**Comment by cbouy** [comment]
Hi,

Most likely the pdb file contains the CONECT record for bonds only for a few non standard residues or small molecules, so since there are bonds MDAnalysis will not try to guess the missing ones.
Try loading the file with `mda.Universe(protein_file, guess_bonds=True)` or have Maestro export all bonds

**Reply by Lili-Cao** [reply]
Hi, 

Yes you are right, the connect info is not for all bonds.
I tried with guess_bond and now its much better.

Thank you!
